### ISSUE TITLE
Accessibility violations in vital components

### ISSUE THREAD

**Issue by uarlouski** [johnsonandjohnson/Bodiless-JS#2130]
## Description

The goal of this issue is to track vital component (at vital-demo site styleguide) violations against WCAG 2.1 AA.

## Components

- Video
  - [aria-allowed-attr](https://dequeuniversity.com/rules/axe/4.7/aria-allowed-attr?application=playwright) violation in link inside video iframe so probably out of scope 
  - [frame-title](https://dequeuniversity.com/rules/axe/4.7/frame-title?application=playwright) violation in video iframe so probably out of scope 
- Layout
  - [list](https://dequeuniversity.com/rules/axe/4.7/list?application=playwright) violation in breadcrumbs
- Generic Template
  - [list](https://dequeuniversity.com/rules/axe/4.7/list?application=playwright) violation in breadcrumbs 
- Section (ok)
- Cards (ok)
- Typography (ok)
- Buttons (ok)
- Accordions (ok)
- Product (ok)

[reports.zip](https://github.com/johnsonandjohnson/Bodiless-JS/files/11735398/reports.zip)

**Comment by perzeababa** [comment]
@uarlouski  can you include the report?

**Comment by hvanyo** [comment]
Can't fix Youtube violations. :(
